README.md

```diff
@@ -75,9 +75,9 @@ If you want to build without Flatpak anyway these instructions might help:
 
 build-time dependencies: `meson, blueprint-compiler, gettext`
 
-runtime dependencies: `matplotlib, python3-matplotlib-gtk4, scipy, numpy`
+runtime dependencies: `matplotlib, python3-matplotlib-gtk4, scipy, numpy, numexpr, sympy`
 
-The actual package names might vary depending on your distribution.
+This actual package names might vary depending on your distribution, and depending on your distribution additional packages may be required.
```

Comment:
Why the change to "This" here?

Reply:
Yeah that's a mistake somehow, not sure why I changed that.

 I remember that I ran into issues with GTK libraries that also had to be installed on Silverblue (which don't come in the Fedora Toolbox), so I thought it makes sense to add the latter part that some additional packages may be required as well. (gtk and libadwaita dev packages mainly). 
 
 Probably wanted to rewrite the sentence a bit differently and started on that, but then decided on appending this new sentence at the end. Either way, the "This" change should be gone, thanks for spotting that.